src/Compilers/CSharp/Test/Symbol/Symbols/Source/EventTests.cs

```diff
@@ -37,7 +37,7 @@ public static void Main()
 
             var actualSymbols = comp.Assembly.GlobalNamespace.GetMembers();
             var actual = string.Join(", ", actualSymbols.Where(s => !s.IsImplicitlyDeclared).Select(symbol => symbol.Name).OrderBy(name => name));
-            Assert.Equal("main1, Test1", actual);
+            Assert.Equal("Test1", actual);
```

Comment:
>Assert.Equal("Test1", actual); [](start = 12, length = 30)

It appears we are consuming ```}``` after the event declaration as part of that declaration. We probably shouldn't do this, it doesn't look like parser did this before. Should also add similar test for scenario when there is no ```;```.

Reply:
This scenario should be fixed.